doc/FEATURES_RELEASE_2.md

```diff
@@ -53,11 +54,86 @@ must display useful error messages about such constructs to the user.
 
 Visit the [homepage] and select the file
 `panacea/test/fixtures/analysis/clashes.pml`. Press `Submit`. A warning should
-now be visible outlining the `PML Construct Name Clashes` in the file.
+now be visible in the warnings tab outlining the `PML Construct Name Clashes` in
+the file.
 
 Specifically, it should highlight name clashes for constructs named `clash1`
 and `clash2`.
 
+## Identify Sequential DDIs - Complete
+
+### Description
+Sequential DDIs arise because the PML contains a workflow in which the
+dispensing of one drug is always followed by the dispensing of another drug that
+has an interaction with the first. The system should identify these sequential
+DDIs and the enclosing `sequence` construct.
+
+### Testing
+Visit the [homepage]. Select `panacea/test/fixtures/ddis.pml` which contains a
+sequential DDI. Press `Submit`. The system should display the DDI information
+for the sequential DDI found in the analysis tab including its enclosing
+construct and the drugs involved.
```

Comment:
Do you it's worth including the specific DDI info that will be visible here? He seems to like a lot of specificity.